Update to Angular 2.4.x

MichaelSolati opened the pull request:
Fixes #4

I bumped the module versions, refactored the component, and tested it on an app I had (it' worked just fine). Mind you I didn't rewrite/update the sample app (only because I'm too lazy).

TheOriginalJosh commented:
Awesome thank you man, I'll merge it this evening. before I do add your self as a contributor to the read me. Thanks agian, I Really appreciate it :)

MichaelSolati commented:
Dude, no, thank you. This package/code was perfect for me. I added a Contributors section to the README and added the other [two people who showed up here](https://github.com/TheOriginalJosh/nativescript-ng2-parallax/graphs/contributors)....
